src/main/java/org/cactoos/iterable/SyncIterable.java

```diff
@@ -58,7 +64,16 @@ public SyncIterable(final X... src) {
      * @param iterable The iterable
      */
     public SyncIterable(final Iterable<X> iterable) {
-        super(new SyncScalar<>(() -> iterable));
+        this.iterable = new UncheckedScalar<>(
```

Comment:
@svendiedrichsen Please, don't use the same attribute and parameter name. I recommend you change `this.iterable` to `this.origin`.